### DISCUSSION TITLE
MQTT - Mosquitto Broker

### DISCUSSION THREAD

**Discussion by home409ca** [homeassistant-apps/app-cloudflared#330] [Q&A]
Hi,
How to issue MQTT on port 1883 via cloudflare?
I am using the add-on: https://github.com/home-assistant/addons/tree/master/mosquitto

Unfortunately I have no idea anymore...

Thank you

**Comment by brenner-tobias** [comment]
Unfortunately, to my knowledge, MQTT is not (yet?) supported at Cloudflared. They are only proxying certain protocols, see [here](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/local/local-management/ingress/#supported-protocols), so you would need to find another way to route your traffic.
